playbooks/common/openshift-master/config.yml

```diff
@@ -337,12 +361,54 @@
     with_items: openshift_master_named_certificates
     when: named_certs_specified | bool
 
-- name: Configure master instances
-  hosts: oo_masters_to_config
+- name: Configure first master instance
```

Comment:
I'm not sure I'm a fan of introducing the duplication here. Granted quite a bit of it will be removed with Andrew's delegate_to changes, but I think the real issue is coming from the attempted use of openshift_version as both an inventory variable and a role variable in this case.

If there is a distinction between the inventory variable and the role variable, then there should be less of an issue with variable dependence and the need to break out the plays separately.

Maybe this is a good place to re-introduce the requested_version variable, only make it the variable that we document and expose through the inventory only. Then it should be possible to handle both the first master and subsequent master use-cases within the same play.